### PR TITLE
Update DynamoWebServer with the latest changes from Benglin/Reach

### PR DESCRIPTION
**Pull request by marimano**
This PR contains changes related to sending array items. And also the changes made by @pboyer in Benglin/Reach.

So after merging all PRs into `DynamoDS/Dynamo:Reaching` this branch will be up-to-date.

@Benglin, please look at it as well and merge in

### CODE REVIEW

File: src/DynamoWebServer/Program.cs [hunk 1]
```diff
@@ -32,9 +32,9 @@ public static void Main(string[] args)
 
             webSocketServer.Start();
 
-            var app = new Application();
-            app.Exit += webSocketServer.ProcessExit;
-            app.Run();
+            Process.GetCurrentProcess().Exited += webSocketServer.ProcessExited;
+
+            while (true) {}
```
Comment (Benglin) [inline src/DynamoWebServer/Program.cs]: I'm sure this is temporary, do we want to do this for the server and enhance this in the next PR?

**Comment by Benglin** [comment]
Hi @marimano, it generally LGTM. If you confirm the above `while (true)` loop is intentional, I can help merging this in for now.

**Comment by marimano** [comment]
Hi @Benglin, those changes in Program.cs are made by @pboyer. So I consider your question should be addressed to him :)

**Comment by Benglin** [comment]
Thanks for clarifying, @marimano! Since @pboyer has done that (probably to unblock some Flood workflows) and he has write access to the repo, I'm going to merge this in and leave the rest to him (I'm confident that he knows what's best for the server side Dynamo instance). LGTM!

**Pull request merged**